From: Shengjing Zhu <zhsj@debian.org>
Date: Mon, 25 Jan 2021 00:47:12 +0800
Subject: disable dhcp test

It needs github.com/d2g/dhcp4server, which is not packaged.

Forwarded: not-needed
---
 plugins/ipam/dhcp/dhcp2_test.go | 2 ++
 plugins/ipam/dhcp/dhcp_test.go  | 2 ++
 2 files changed, 4 insertions(+)

diff --git a/plugins/ipam/dhcp/dhcp2_test.go b/plugins/ipam/dhcp/dhcp2_test.go
index f5a4c32..f3ef5b9 100644
--- a/plugins/ipam/dhcp/dhcp2_test.go
+++ b/plugins/ipam/dhcp/dhcp2_test.go
@@ -1,3 +1,5 @@
+// +build ignore
+
 // Copyright 2015-2018 CNI authors
 //
 // Licensed under the Apache License, Version 2.0 (the "License");
diff --git a/plugins/ipam/dhcp/dhcp_test.go b/plugins/ipam/dhcp/dhcp_test.go
index 2a1de5c..06cc97a 100644
--- a/plugins/ipam/dhcp/dhcp_test.go
+++ b/plugins/ipam/dhcp/dhcp_test.go
@@ -1,3 +1,5 @@
+// +build ignore
+
 // Copyright 2015-2018 CNI authors
 //
 // Licensed under the Apache License, Version 2.0 (the "License");
